src/utils/options.js

```diff
@@ -7,6 +7,7 @@ function getOption({ opts }, name, defaultValue = true) {
 export const useDisplayName = state => getOption(state, 'displayName')
 export const useSSR = state => getOption(state, 'ssr', true)
 export const useFileName = state => getOption(state, 'fileName')
+export const useFolderName = state => getOption(state, 'folderName', false)
```

Comment:
I think this could be named more obviously... perhaps `prefixComponentWithFolderName`

Reply:
`folderName` sounds not less clearer as `fileName`. Context matters.